sticky_notes.py

```diff
@@ -43,6 +43,23 @@ def sender(x):
                 while len(notes) >= 1:
                     note = notes.pop()
                     self.delay_task(len(notes),
-                                    sender(format_note(nick, note)))
+                                    self.sender(format_note(nick, note), event))
                 self.notes[nick] = []
                 return format_note(nick, last_note)
+
+    @cmd
+    def list_stickynotes(self, event):
```

Comment:
@JakubNvk this is a command, it should have a docstring so that a help message can be generated.